build/publish/PublishDebian.targets

```diff
@@ -19,7 +19,7 @@
     </DebianUploadJsonContent>
   </PropertyGroup>
 
-  <Target Name="PublishDebFilesToDebianRepo" Condition=" '$(OSName)' == 'ubuntu' AND '$(SkipPublishToDebianRepo)' != 'true' ">
+  <Target Name="PublishDebFilesToDebianRepo" Condition=" '( $(OSName)' == 'ubuntu' OR '$(OSName)' == 'debian' ) AND '$(SkipPublishToDebianRepo)' != 'true' ">
```

Comment:
Just checking, does something need to change somewhere such that REPO_ID is set to something debian specific for the debian build (maybe this happens in official build definitions or something)?